tf-modules/distribution/variables.tf

```diff
@@ -1,5 +1,9 @@
 # Required
 
+variable "lambda_processing_role_arn" {
+  type        = string
+  description = "Cumulus lambda processing role"
+}
```

Comment:
nit: follow with a blank line for consistency

Reply:
9c01487d4

Reply:
0a3d84995